changelog.dd

```diff
@@ -28,12 +28,12 @@ $(BUGSTITLE Library Changes,
         exposed))
     $(LI $(RELATIVE_LINK2 iota-length-size_t, `std.range.iota's `.length`
         property is fixed to `size_t` instead of the type being iterated))
-    $(LI $(XREF uni, isNumber) and $(XREF uni, isPunctuation) now use a separate,
+    $(LI $(REF isNumber, std,uni) and $(REF isPunctuation, std,uni) now use a separate,
```

Comment:
yes REF is not defined in the changelog anymore - however it is in the dlang.org.ddoc.
How about the following?

```
dmd -D changelog.dd -D ../dlang.org/dlang.org.ddoc -D ../dlang.org/doc.ddoc -D ../dlang.org/html.ddoc -D ../dlang.org/std.ddoc 
```